docker/Dockerfile

```diff
@@ -7,19 +7,21 @@ COPY ./go-tf-prepare/main.go main.go
 COPY ./go-tf-prepare/pkg/ pkg/
 RUN GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o tf-prepare main.go
 
-FROM alpine:3.12.1
+FROM debian:bullseye-slim
+# 11.1-slim
+
+#Base
+RUN apt update && \
+    apt install -y git curl openssl && \
+    apt install -y pip && \
+    apt install -y gcc libffi-dev libssl-dev python3-dev make unzip wget && \
+    pip --no-cache-dir install -U pip && \
+    apt-get install ansible && \
+    apt-get install packer
```

Comment:
Define which  version of ansible & packer that we are using.

Reply:
Added version.